Profile Folder/chrome/locale/ru-RU/dtd/gsettings.dtd

```diff
@@ -145,13 +145,13 @@
 <!ENTITY gsettings.themeToBehaviour             "Настройте, насколько сильно темы влияют на интерфейс...">
 <!ENTITY gsettings.themeToExperiments           "Персонализируйте дальше с помощью заброшенных опций Chromium...">
 
-<!ENTITY gsettings.peoplePositionTitle			"Позиция">
+<!ENTITY gsettings.peoplePositionTitle			"Позиция кнопки профиля">
 <!ENTITY gsettings.peoplePositionDescription	"На протяжении многих лет кнопка Профиль занимала разные позиции.">
-<!ENTITY gsettings.tabsToolbar                  "[!!! Táβƨ Tôôℓβář !!!]">
-<!ENTITY gsettings.titlebarButtons              "[!!! Tïƭℓèβář βúƭƭôñƨ !!!]">
+<!ENTITY gsettings.tabsToolbar                  "На панели вкладок">
+<!ENTITY gsettings.titlebarButtons              "Возле кнопок панели вкладок">
 
-<!ENTITY gsettings.oldButton		"[!!! Óℓδ βúƭƭôñ !!!]">
```

Comment:
I am so sorry for this but last night I didn't verify the strings properly because I had to go to bed and ended up sending out an outdated one :(
https://github.com/angelbruni/Geckium/commit/66cd45532db9c148debc7f9d13333f5c258dd0bd updates it.
Once again, I am sorry.

Reply:
done